docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
+
+   ```
+    $ kubectl get pod pod1
+    $ kubectl get pods pod1
+   ```
+
+* `NAME`: Specifies the name of the resource. Names are case-sensitive. If the name is omitted, details for all resources are displayed, for example `$ kubectl get pods`.
+
+   When performing an operation on multiple resources, you can specify each resource by type and name or specify one or more files:
+   * To specify resources by type and name:
+	    * To group resources if they are all the same type: `TYPE1 name1 name2 name<#>`<br/>
+        Example: `$ kubectl get pod example-pod1 example-pod2`
+        * To specify multiple resource types individually: `TYPE1/name1 TYPE1/name2 TYPE2/name3 TYPE<#>/name<#>`<br/>
+        Example: `$ kubectl get pod/example-pod1 replicationcontroller/example-rc1`
+   * To specify resources with one or more files: `-f file1 -f file2 -f file<#>`
+	 [Use YAML rather than JSON](config-best-practices.md) since YAML tends to be more user-friendly, especially for configuration files.<br/>
+     Example: `$ kubectl get pod -f ./pod.yaml`
+* `flags`: Specifies optional flags. For example, you can use the `-s` or `--server` flags to specify the address and port of the Kubernetes API server.<br/>
+**Important**: Flags that you specify from the command line override default values and any corresponding environment variables. *Tip*: Use short flags sparingly and only for the most frequently used options.<br/>
+For more information and examples about specifying flags, see the [kubectl conventions](../devel/kubectl-conventions.md#flag-conventions).
+
+If you need help, just run `kubectl help` from the terminal window.
+
+## Operations
+
+The following table includes short descriptions and the general syntax for all of the `kubectl` operations:
+
+Operation       | Syntax	|       Description
+-------------------- | -------------------- | --------------------
+`annotate`	| `kubectl annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]` | Add or update the annotations of a resource.
+`api-versions`	| `kubectl api-versions` | List the API versions in use.
+`apply`			| `kubectl apply` | Apply a configuration change to a resource by file or stdin.
+`attach`		| `kubectl attach POD -c CONTAINER` | Attach to a running container to send stdin from the terminal window.
+`cluster-info`	| `kubectl cluster-info` | Display endpoint information about the master and services in the cluster.
+`config`		| `kubectl config SUBCOMMAND` | Modifies kubeconfig files. See the sub-commands for details.
+`create`		| `kubectl create -f FILENAME` | Create a resource using a file or stdin.
+`delete`		| `kubectl delete ([-f FILENAME] | TYPE [(NAME | -l label | --all)])` | Delete resources either by filenames, label selectors, names, resource selectors, resources, or stdin.
```

Comment:
change the syntax.

**Question**
I changed this description to align with all "-f FILENAME" descriptions, is it accurate:
"Delete resources either from a file, stdin, or specifying label selectors, names, resource selectors, or resources."